.github/workflows/android-perf.yml

```diff
@@ -0,0 +1,179 @@
+name: android-perf
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  pull_request:
+  push:
+    tags:
+      - ciflow/perf-android
+  # Note: GitHub has an upper limit of 10 inputs
+  workflow_dispatch:
```

Comment:
Please keep in mind that GitHub, for some reason, only allows up to 10 parameters when using `workflow_dispatch`.  I learn about that limit the hard way :)